### PR TITLE
documentation updates

### PR DESCRIPTION
**Pull request by holzerma**
## Type of request

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring 
- [x] Documentation or documentation changes

#### Description of the change
updated some basic documents for the start of the open source project

#### Initiator
Matthias Holzer / Allianz Technology GmbH 

## Checklist:

- [x] My code follows the code style of this project.
- [x] I aggree with die CLA.
- [x] I have read the CONTRIBUTING docs.
- [x] I have added/updated necessary documentation (if appropriate).

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -34,13 +34,19 @@ This Code of Conduct applies both within project spaces and in public spaces whe
...
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at hello@aposin.org. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
 
 Project maintainers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership.
 
+## Escalation
+
+To report an issue with the project leadership team, please contact Matthias Holzer (@holzerma) or Markus Rothmueller (@MarRothm) via GitHub.
```
Comment (d-gs) [inline CODE_OF_CONDUCT.md]: Add links to the user names.

```suggestion
To report an issue with the project leadership team, please contact Matthias Holzer ([@holzerma](https://github.com/holzerma)) or Markus Rothmueller ([@MarRothm](https://github.com/holzerma)) via GitHub.
```

File: CODING_MANIFEST.md [hunk 2]
```diff
@@ -1,40 +1,36 @@
+
+
+
```
Comment (d-gs) [inline CODING_MANIFEST.md:3]: Delete extra new lines

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -34,13 +34,19 @@ This Code of Conduct applies both within project spaces and in public spaces whe
...
+
 ## Attribution
 
 This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4, available at [http://contributor-covenant.org/version/1/4][version]
 
 [homepage]: http://contributor-covenant.org
 [version]: http://contributor-covenant.org/version/1/4/
+
+
```
Comment (d-gs) [inline CODE_OF_CONDUCT.md:52]: Delete extra new lines

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,3 +1,6 @@
+
+
+
```
Comment (d-gs) [inline CONTRIBUTING.md:3]: Delete extra new lines

**Review by d-gs — CHANGES_REQUESTED** [review]
Some changes to be done. Then LGTM

**Comment by holzerma** [comment]
updated your input.

**Review by d-gs — APPROVED** [review]

**Pull request merged**